Pickle-Finance-Agents/RangeManagementMonitor/src/mock.provider.ts

```diff
@@ -0,0 +1,42 @@
+import { Interface } from "@ethersproject/abi";
```

Comment:
I think you are not using this file

Reply:
This one is used in the `addCallTo` function definition

Reply:
Oh, I misread your comment, the file is used too, in `strategy.fetcher.spec.ts`